### PR TITLE
chore: Minor code refactor adhering to ES6 semantics

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
refactoring

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
N/A

**Summary**
Use object destructuring assignment wherever possible.

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

File: lib/commands/external.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ class ExternalCommand {
                 choices: ['Yes', 'No', 'Y', 'N', 'y', 'n'],
             },
         ]);
-        if (answer.installConfirm === true) {
+        if (installConfirm) {
```
Comment (evenstensberg) [inline lib/commands/external.js]: Is the var defined for no as well? Reason for having strict true there

File: lib/groups/help.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const commandLineUsage = require('command-line-usage');
 class HelpGroup {
     outputHelp(command = null) {
         if (command !== null) {
-            const commands = require('../utils/commands').info;
+            const { commands } = require('../utils/commands');
```
Comment (hemal7735) [inline lib/groups/help.js]: this line changes the meaning.
should not it be `const { info } = require('../utils/commands');`

File: lib/utils/commands.js [hunk 3]
```diff
@@ -9,5 +9,7 @@ const names = commands
     })
     .reduce((arr, val) => arr.concat(val), []);
 
-module.exports.names = names;
-module.exports.info = commands;
+module.exports = {
```
Comment (hemal7735) [inline lib/utils/commands.js]: reassigning the `exports` object should be avoided.
Please revert this change.

Comment (jamesgeorge007) [reply lib/utils/commands.js]: Changed the named export to `module.exports.commands = commands;`

File: lib/groups/help.js (hunk 2 above)
Comment (jamesgeorge007) [reply lib/groups/help.js]: Changed the named export to `module.exports.commands = commands;`

**Comment by hemal7735** [comment]
@jamesgeorge007 there are many test cases are failing. This could be due to the chain effect of not updating the test-cases based on the changes here.
Please run the test-cases in local to see what failed.

**Comment by jamesgeorge007** [comment]
@hemal7735 CI couple of recent works in the `next` branch made the CI fail which has nothing to do with this PR. Check out the [CI build status](https://travis-ci.org/webpack/webpack-cli/).

File: lib/utils/commands.js [hunk 4]
```diff
@@ -9,5 +9,5 @@ const names = commands
     })
     .reduce((arr, val) => arr.concat(val), []);
 
+module.exports.commands = commands;
```
Comment (evenstensberg) [inline lib/utils/commands.js]: ```module.exports = { commands, names }```

Comment (jamesgeorge007) [reply lib/utils/commands.js]: https://github.com/webpack/webpack-cli/pull/1122#discussion_r347212792

Comment (jamesgeorge007) [reply lib/utils/commands.js]: https://github.com/webpack/webpack-cli/pull/1122#discussion_r347212792

File: lib/groups/help.js (hunk 2 above)
Comment (ematipico) [reply lib/groups/help.js]: Let's go with `commands`. `info` doesn't help at all as name. Good change, thank you!

**Comment by jamesgeorge007** [comment]
@ematipico hope we're good to land :rocket:

**Comment by rishabh3112** [comment]
@jamesgeorge007 have a look at #1141. There may be some duplicate changes here.
